wikidev/wikidev/views.py

```diff
@@ -28,68 +34,82 @@ def get_user(self):
         data = jwt.decode(token.replace('Bearer ', ''), key='87654321')
         user, created = User.objects.get_or_create(id=data['sub'],
                                                    username=str(data['sub']))
-       # user, created = User.objects.get_or_create(id=231243,
-       #                                            username='Mircea')
         return user
 
-    def to_response(self, menu_or_form):
-        response = onem.Response(
-                menu_or_form, self.request.headers['X-Onem-Correlation-Id']
-        )
-        return HttpResponse(response.as_json(), content_type='application/json')
+    def to_response(self, content):
+        response = Response(content=content)
+        response.correlation_id = self.request.headers['X-Onem-Correlation-Id']
```

Comment:
Let's drop this line as `correlation_id` will be removed from Response in the next version.